lib/URI.pm

```diff
@@ -571,8 +572,12 @@ removing the explicit port specification if it matches the default port,
 uppercasing all escape sequences, and unescaping octets that can be
 better represented as plain characters.
 
-For efficiency reasons, if the $uri is already in normalized form,
-then a reference to it is returned instead of a copy.
+Up to version 1.74, this method would return the original unchanged
```

Comment:
I'd change this to "Before version 1.75", as that's the version users would need to depend on if they want/need the new behaviour.